manifests/plugin/openscap.pp

```diff
@@ -0,0 +1,31 @@
+# = Foreman Proxy OpenSCAP plugin
+#
+# This class installs OpenSCAP plugin
+#
+# === Parameters:
+#
+# $enabled::                    enables/disables the plugin
+#                               type:boolean
+# $openscap_send_log_file::     Log file for the forwarding script
+#
+# $spooldir::                   Directory where OpenSCAP audits are stored
+#                               before they are forwarded to Foreman
+class foreman_proxy::plugin::openscap (
+  $enabled                = $::foreman_proxy::plugin::openscap::params::enabled,
+  $openscap_send_log_file = $::foreman_proxy::plugin::openscap::params::openscap_send_log_file,
+  $spooldir               = $::foreman_proxy::plugin::openscap::params::spooldir,
+) inherits foreman_proxy::plugin::openscap::params {
+  validate_bool($enabled)
+
+  case $::osfamily {
```

Comment:
Since there's no OS-specific logic I'd be in favour of removing the OS-specific logic.